espnet2/enh/espnet_model.py

```diff
@@ -27,30 +45,28 @@ def __init__(
         self.enh_model = enh_model
```

Comment:
We can't access any methods and any attributes if the abstract class doesn't have.

In python, we can't declare abstract attributes in abstract class, so we need to define `get_num_spk()` for example.
Another way is giving all information as arguments for __init__(). I think the latter is better.

```python
def __init__(self, num_spk, num_noise_type, mask_type, ...)
```

Reply:
Refactored, now it's an abstract proproty of  ABSSeparator.